### PR TITLE
[PasswordHasher] Fix: Use algorithm instead of algo

### PR DESCRIPTION
**Pull request by localheinz**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | n/a
| License       | MIT
| Doc PR        | n/a

This pull request

* [x] renames fields, variables, and parameters using `algos` or `algo` (which appear to be entirely made-up words) to `algorithms` and `algorithm` respectively

### CODE REVIEW

File: src/Symfony/Component/PasswordHasher/Hasher/NativePasswordHasher.php [hunk 1]
```diff
@@ -107,6 +107,6 @@ public function verify(string $hashedPassword, string $plainPassword): bool
      */
     public function needsRehash(string $hashedPassword): bool
     {
-        return password_needs_rehash($hashedPassword, $this->algo, $this->options);
+        return password_needs_rehash($hashedPassword, $this->algorithm, $this->options);
```
Comment (localheinz) [inline src/Symfony/Component/PasswordHasher/Hasher/NativePasswordHasher.php:110]: Yes, `password_needs_rehash()` has a parameter that is named `$algo`, but that does not mean that we need to use the same naming, does it?

Comment (chalasr) [reply src/Symfony/Component/PasswordHasher/Hasher/NativePasswordHasher.php:110]: I don't think so. Let's use true words

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you @localheinz.

**Comment by localheinz** [comment]
Thank you, @chalasr!

**Pull request merged**